packages/frontend/src/pages/SignUp.styles.tsx

```diff
@@ -90,12 +91,31 @@ export const FooterCta = styled.a`
     color: ${Colors.BLUE3};
 `;
 
+export const LinkButton = styled.button`
+    color: ${Colors.BLUE3};
+    border: none;
+    background-color: transparent;
+
+    &:hover {
+        cursor: pointer;
+        text-decoration: underline;
+    }
+`;
+
 export const Subtitle = styled.p`
     color: ${Colors.GRAY2};
     margin: 5px 0 20px;
     text-align: center;
 `;
 
+export const ErrorMessage = styled.p`
+    color: ${Colors.RED3};
+    font-size: 12px;
+    font-weight: 500;
+    margin: 5px 30px 15px;
+    text-align: center;
+`;
```

Comment:
FormGroup components has error state so you don't need any of that
![CleanShot 2023-03-02 at 16 45 52@2x](https://user-images.githubusercontent.com/962095/222432655-2dd436c5-e096-4d2b-a777-ec44e2386a3c.png)